src/main/java/in/twizmwaz/cardinal/command/TimeLimitCommand.java

```diff
@@ -18,8 +21,11 @@
     @Command(aliases = {"timelimit", "tl"}, desc = "Modify the time limit of the current match.", usage = "<add, set> <time> [result]")
     public static void timeLimit(final CommandContext cmd, CommandSender sender) throws CommandException {
         if (cmd.argsLength() == 0) {
-            if (TimeLimit.getMatchTimeLimit() != 0) {
+            Match match = GameHandler.getGameHandler().getMatch();
+            if ((TimeLimit.getMatchTimeLimit() != 0) && (!(match.getState().equals(MatchState.ENDED)) || (match.getState().equals(MatchState.CYCLING)))) {
```

Comment:
@iEli2tyree011 If a player do ```/timelimit``` when the match is starting or running, it will say the actual time limit, on the other hand, if the command has been executed when the match is cycling or end, the message will report the time that last the match. Already tested, but I'm very busy to test again right now or send you an image, sorry. Although, you can test it